### DISCUSSION TITLE
Youtube does not process exported videos and even Windows Video Editor gives error.

### DISCUSSION THREAD

**Discussion by Dagg3rFall** [mifi/lossless-cut#1155] [Q&A]
So after a day of uploading many videos, parts of videos, split etc on youtube, all videos gave processing errors no matter how small or how big a file size is. The videos upload but do not process.

I also imported videos edited in Lossless Cut to Windows video editor under Photos app. It straight away gave this error. "files have properties that can not be read" confirming its not my youtube account or my video files but the software giving issues.

All my videos have a coverart attached inside Lossless Cut.

**Comment by rd-sf** [comment]
Same/Similar here.  I took two GoPro videos (MP4) shot sequentially on the same GoPro with the same settings, used "Merge/concatenate files..." and YouTube errors when processing the upload.  When I play back the concatenated file the video freezes at the point where it would have switched files.  I would share the files, but I deleted the originals before checking the full (LONG) output file.  Oops.

**Reply by mifi** [reply]
Hard to say what's the reason without having the files in question, but yea maybe LosslessCut should make it more clear that the user should not delete original files until 100% sure that the output works
